Adapter/SymfonyCache.php

```diff
@@ -65,8 +70,9 @@ class SymfonyCache implements CacheAdapterInterface
      * @param bool            $phpCodeCacheEnabled If true, will clear APC or PHP OPcache code cache
      * @param array           $types               A cache types array
      * @param array           $servers             An array of servers
+     * @param array           $timeout             An array of timeout options
      */
-    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers)
+    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers, array $timeout)
```

Comment:
There are more and more arguments here some of which are options, the constructor is getting big… maybe we should consider using the options resolver (in another PR)?